src/plugins/prebuilt-library/data-fetch/get-async-data.js

```diff
@@ -365,7 +365,7 @@ export function getAsyncData() {
 					key: key ? key : library,
 					pattern_id: item_id ? item_id : '',
 					pattern_type: type ? type : '',
-					pattern_type: style ? style : '',
+					pattern_style: style ? style : '',
```

Comment:
oh nice, I added a TODO in my branch about this, wasn't sure how to fix it until now.